ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -26,6 +26,7 @@ function mirror_test_images() {
 
         # shellcheck disable=SC2087
         ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF
+set -o errexit
```

Comment:
This is already set at the entry point of the file, why is it needed here? Is `oc` not returning an error correctly on failures?

Reply:
The inlined bash script doesn't inherit the top-level declaration and execution within that script will continue despite errors.  Mirroring errors will be suppressed/masked.